Spanner/src/Operation.php

```diff
@@ -388,6 +439,12 @@ public function read(Session $session, $table, KeySet $keySet, array $columns, a
         $context = $this->pluck('transactionContext', $options);
 
         $call = function ($resumeToken = null) use ($table, $session, $columns, $keySet, $options) {
+            if (
+                isset($options['transactionHandle']) &&
+                !is_null($options['transactionHandle']->id())
```

Comment:
I don't think we should be passing around an option like `transactionHandle`. See this PR for how this can be cleaned up: https://github.com/googleapis/google-cloud-php/pull/6810

Reply:
Implemented alternative2 resolve this comment.